org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/FoldingConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package org.eclipse.xtext.testing;
+
+public class FoldingConfiguration extends TextDocumentConfiguration {
+
+	private String expectedFoldings;
+
+	public String getExpectedFoldings() {
+		return expectedFoldings;
+	}
+
+	public void setExpectedFoldings(String expectedFoldings) {
+		this.expectedFoldings = expectedFoldings;
```

Comment:
Question: Don't we expect multiple foldings?

Reply:
We do, but I just took the same approach as the [completion testing](https://github.com/eclipse/xtext-core/blob/2027567c0115cda8a6dd071f44ebb7615640991e/org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/CompletionTest.java#L52-L63).